extension/wrapper.cpp

```diff
@@ -144,6 +168,8 @@ Satrec__sgp4(PyObject *self, PyObject *args)
 static PyMethodDef Satrec_methods[] = {
     {"twoline2rv", (PyCFunction)Satrec_twoline2rv, METH_VARARGS | METH_CLASS,
      PyDoc_STR("Initialize the record from two lines of TLE text.")},
+    {"sgp4init", (PyCFunction)Satrec_sgp4init, METH_VARARGS | METH_CLASS,
+     PyDoc_STR("Initialize the record directly from provided variables.")},
```

Comment:
Let's be a tiny bit more specific: `Initialize the record from orbital elements.`